### DISCUSSION TITLE
🚗 Добавить зимнюю резину

### DISCUSSION THREAD

**Discussion by damirmd** [octothorp-team/public-dobrograd#809] [Предложения]
Предлагаю когда добавят зимнюю карту сделать такой предмет на машину как '**зимняя резина**'. Его нужно будет покупать отдельно в магазине и устанавливать на зимний срок. Можно реализовать, что если не ставить зимнюю резину – машину будет заносить в поворот и ей неудобно будет управлять.

**Comment by Hartsig** [comment]
Еу ет че зимний дрифт намечается?
